handbook/company/product-groups.md

```diff
@@ -540,7 +543,7 @@ PRs should remain in draft until they are ready to be reviewed for final approva
 All PRs from the community are routed through the on-call developer. For code changes, if the on-call developer has the knowledge and confidence to review, they should do so. Otherwise, they should request a review from an developer with the appropriate domain knowledge. It is the on-call developer's responsibility to monitor community PRs and make sure that they are moved forward (either by review with feedback or merge).
 
 - **Customer success meetings**
-The on-call developer is encouraged to attend some of the customer success meetings during the week. Post a message to the #g-endpoint-ops Slack channel requesting invitations to upcoming meetings.
+The on-call developer is encouraged to attend some of the customer success meetings during the week. Post a message to the #g-endpoint-o Slack channel requesting invitations to upcoming meetings.
```

Comment:
Should we change to `#help_engineering`?

Reply:
This should actually be #g-customer-success.